torch/onnx/symbolic_opset9.py

```diff
@@ -471,8 +471,17 @@ def expand_as(g, self, other):
     return g.op("Expand", self, shape)
 
 
+@parse_args('v', 'v', 'i', 'v', 'v')
 def embedding(g, weight, indices, padding_idx, scale_grad_by_freq, sparse):
```

Comment:
@Derek pointed out that we are not handling the `scale_grad_by_freq` argument, which affects how gradients are scaled in the backward pass for this op. That is relevant for training. What can we do about that? 
At the very least, we can check if the export is in training mode and `scale_grad_by_freq=True`, and error out or warn. Let's think about that.

Reply:
Addressed.